vercel/turbo
# feat(turbopack): Lint invalid CSS modules

### Description

`css-loader` seems to have a linting rule for invalid CSS Modules.
This PR implements same rule for `swc_css` and `lightningcss`.

### Testing Instructions

See next.js counterpart: https://github.com/vercel/next.js/pull/62040
Closes PACK-2408
